manifests/cf-rabbitmq-server-only-template.yml

```diff
@@ -46,6 +46,8 @@ properties:
     - rabbitmq_management
     - rabbitmq_mqtt
     - rabbitmq_stomp
+    policies:
+      HA: '''^(?!amq\.).*'' ''{"ha-mode": "all"}'''
```

Comment:
@axelaris I'll defer to the team that maintains this release but I personally would consider two alternatives:

 * Using a comment to demonstrate how the property is used
 * Mirroring to 2 nodes instead of `all` (which may or may not be necessary with cluster size of >= 3 and is excessive with cluster size of >= 4)

A combination of both would probably be even better.